docs/modules/ROOT/pages/api/governance.adoc

```diff
@@ -10,11 +10,1977 @@
+
+Runs the already queued proposal through the timelock.
+
+[.contract-item]
+[[GovernorTimelockExecutionComponent-queue_operations]]
+==== `[.contract-item-name]#++queue_operations++#++(ref self: ContractState, proposal_id: felt252, calls: Span<Call>, description_hash: felt252) → u64++` [.item-kind]#internal#
+
+Queue a proposal to the timelock.
+
+Returns the eta for the execution of the queued proposal.
+
+[.contract-item]
+[[GovernorTimelockExecutionComponent-proposal_needs_queuing]]
+==== `[.contract-item-name]#++proposal_needs_queuing++#++(self: @ContractState, proposal_id: felt252) → bool++` [.item-kind]#internal#
+
+In this implementation, it always returns true.
+
+[.contract-item]
+[[GovernorTimelockExecutionComponent-cancel_operations]]
+==== `[.contract-item-name]#++cancel_operations++#++(ref self: ContractState, proposal_id: felt252, description_hash: felt252)++` [.item-kind]#internal#
+
+Cancels the timelocked proposal if it has already been queued.
+
+[#GovernorTimelockExecutionComponent-Embeddable-Functions]
+==== Embeddable functions
+
+:TimelockUpdated: xref:#GovernorTimelockExecutionComponent-TimelockUpdated[TimelockUpdated]
+
+[.contract-item]
+[[GovernorTimelockExecutionComponent-timelock]]
+==== `[.contract-item-name]#++timelock++#++(self: @ContractState) → ContractAddress++` [.item-kind]#external#
+
+Returns the timelock controller address.
+
+[.contract-item]
+[[GovernorTimelockExecutionComponent-get_timelock_id]]
+==== `[.contract-item-name]#++get_timelock_id++#++(self: @ContractState) → felt252++` [.item-kind]#external#
+
+Returns the timelock proposal id for a given proposal id.
+
+[.contract-item]
+[[GovernorTimelockExecutionComponent-update_timelock]]
+==== `[.contract-item-name]#++update_timelock++#++(ref self: ContractState, new_timelock: ContractAddress)++` [.item-kind]#external#
+
+Updates the associated timelock.
+
+Requirements:
+
+- The caller must be the governance.
+
+Emits a {TimelockUpdated} event.
+
+[#GovernorTimelockExecutionComponent-Internal-Functions]
+==== Internal functions
+
+[.contract-item]
+[[GovernorTimelockExecutionComponent-initializer]]
+==== `[.contract-item-name]#++initializer++#++(ref self: ContractState, timelock: ContractAddress)++` [.item-kind]#internal#
+
+Initializes the timelock controller.
+
+Requirements:
+
+- The timelock must not be the zero address.
+
+[.contract-item]
+[[GovernorTimelockExecutionComponent-assert_only_governance]]
+==== `[.contract-item-name]#++assert_only_governance++#++(self: @ContractState)++` [.item-kind]#internal#
+
+Ensures the caller is the executor (the timelock controller in this case).
+
+[.contract-item]
+[[GovernorTimelockExecutionComponent-timelock_salt]]
+==== `[.contract-item-name]#++timelock_salt++#++(self: @ContractState, description_hash: felt252) → felt252++` [.item-kind]#internal#
+
+Computes the `TimelockController` operation salt.
```

Comment:
```suggestion
Computes the `TimelockController` operation salt as the XOR of the governor address and `description_hash`.
```
In case users are curious  🤷‍♂️